pkg/bpf/exporter.go

```diff
@@ -142,7 +145,7 @@ func (e *exporter) attach() error {
 	}
 
 	// Return early if hardware counters are not enabled
-	if !config.ExposeHardwareCounterMetrics() {
+	if !config.IsExposeHardwareCounterMetrics() {
```

Comment:
Same as above, lets not change it. 
`config.ExposeHwCounterMetrics()` reads better (proper English) than `config.IsExposeHardwareCounterMetrics`